legend-engine-xt-relationalStore-pure/src/main/resources/core_relational/relational/relationalExtension.pure

```diff
@@ -86,9 +86,8 @@ function meta::relational::runtime::dataSourceEquality(a:DataSource[1],b:DataSou
 }
 
 
-function meta::relational::functions::typeInference::inferDynaFunctionReturnType(d: DynaFunction[1], failOnMatchFailure:Boolean[1]):meta::relational::metamodel::datatype::DataType[0..1]
```

Comment:
what if this fn is used somewhere (it is public), can you instead maintain the original version besides the overridden version with dfToReturnFunctionMap

Reply:
sure